run_anvil.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
related to the other comment, if we have the flags in the docker-compose, we can remove this.